File: src/ZKWeb.MVVMPlugins/MVVM.Common.Organization/src/Application/Dtos/RoleOutputDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using ZKWeb.MVVMPlugins.MVVM.Common.Base.src.Application.Dtos;

namespace ZKWeb.MVVMPlugins.MVVM.Common.Organization.src.Application.Dtos {
	[Description("角色传出信息")]
	public class RoleOutputDto : IOutputDto {
		[Description("角色Id")]
		public Guid Id { get; set; }
		[Description("角色名称")]
		public string Name { get; set; }
		[Description("权限列表")]
		public IList<string> Privileges { get; set; }
		[Description("租户名")]
		public string OwnerTenantName { get; set; }
		[Description("租户Id")]
		public Guid OwnerTenantId { get; set; }
		[Description("创建时间")]
		public string CreateTime { get; set; }
		[Description("更新时间")]
		public string UpdateTime { get; set; }
		[Description("备注")]
		public string Remark { get; set; }
		[Description("已删除")]
		public bool Deleted { get; set; }

		public RoleOutputDto() {
			Privileges = new List<string>();
		}
	}
}

File: src/ZKWeb.MVVMPlugins/MVVM.Angular.Website/static.old/src/modules/generated_module/dtos/role-output-dto.ts
/** 角色传出信息 */
export class RoleOutputDto {
    /** 角色Id */
    public Id: string;
    /** 角色名称 */
    public Name: string;
    /** 权限列表 */
    public Privileges: string[];
    /** 权限名称列表 */
    public PrivilegeNames: string;
    /** 租户名 */
    public OwnerTenantName: string;
    /** 租户Id */
    public OwnerTenantId: string;
    /** 创建时间 */
    public CreateTime: string;
    /** 更新时间 */
    public UpdateTime: string;
    /** 备注 */
    public Remark: string;
    /** 已删除 */
    public Deleted: boolean;
}
